Clinical-Genomics/cgweb
# Added qc sample field

How to test:
0. Install branch on stage on clinical-db
1. login 
2. new order
3. Add microbial samples file
4. Save order
5. Submit order
Expected outcome:
The field proceed only if qc ok in lims is the same as the value in the excel file
